sql/build.gradle

```diff
@@ -10,6 +10,10 @@ configurations {
     benchmarksRuntime.extendsFrom testRuntime, benchmarksCompile
 }
 
+repositories {
+    jcenter()
```

Comment:
it seems to be only the place where jcenter is needed. But we can put it also [here](https://github.com/crate/crate/blob/master/gradle/javaModule.gradle) as it is done to jitpack repo. would it be ok?